ui/app/AppLayouts/Communities/stores/CommunitiesStore.qml

```diff
@@ -230,6 +237,14 @@ QtObject {
         return communitiesModuleInst.isDisplayNameDupeOfCommunityMember(displayName)
     }
 
+    function leaveCommunity(communityId) {
+        d.profileSectionModuleInst.communitiesModule.leaveCommunity(communityId)
+    }
+
+    function setCommunityMuted(communityId, mutedType) {
+        d.profileSectionModuleInst.communitiesModule.setCommunityMuted(communityId, mutedType)
```

Comment:
Same thing

Reply:
Actually, if we wanted a very thorough cleanup, we could remove those two function from the chat section as well and rely on the communities module instead. We just have to pass the `communityId`